tools/check-openapi

```diff
@@ -16,6 +17,10 @@ const SwaggerParser = require('swagger-parser');
                 }).openapi !== undefined
             ) {
                 await SwaggerParser.validate(file);
+                const res = await ExampleValidator.validateFile(file);
+                if (res.valid !== true) {
+                    throw new Error("One or more examples are invalid.");
+                }
```

Comment:
The library gives us a list of errors, might as well use it.

```suggestion
                if (!res.valid) {
                    for (const error of res.errors) {
                        console.error(error);
                    }
                    process.exitCode = 1;
                }
```

Reply:
Changed accordingly.